tools/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -29,21 +29,18 @@ or logos must follow Microsoft's [Trademark & Brand Guidelines](https://www.micr
 Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
 `;
 
+// Packages which definitely should not publish on NPM
 function packageShouldBePrivate(name: string): boolean {
-    // allow test packages to be packaged
-    if (name.startsWith("@fluid-internal/test-")) {
```

Comment:
As mentioned in chat, we need this so they can publish to the internal feed for CI stuff.

Reply:
Reverted in latest, will follow up to better understand this dependency.